### ISSUE TITLE
No 'num_batches_tracked' in state_dict

### ISSUE THREAD

**Issue by bonlime** [mapillary/inplace_abn#130]
Native torch BatchNorm has  `num_batches_tracked` in it's state_dict, while ABN doesn't. It makes loading weights trained with usual BN+Activation inconvenient. Is there any reason why you didn't add this item into the dict?

**Comment by ducksoup** [comment]
The reason why there's no `num_batches_tracked` is that we don't support the "cumulative moving average" mode, i.e. `momentum=None` in Pytorch's standard BN layer, introduced in Pytorch 1.0. Support for loading BN weights from both pre-1.0 (i.e. without the `num_batches_tracked` parameter) and post-1.0 models is not straightforward, but we are working on a fix.

In the meanwhile, a simple work-around for you should be to pass `strict=False` as an additional parameter to `load_state_dict()`.

**Comment by ducksoup** [comment]
I just found an easier solution to this issue, please update to `v1.0.4` for a fix.

**Comment by bonlime** [comment]
Thx for a quick fix!
